src/EnergyPlus/GeneratorFuelSupply.cc

```diff
@@ -152,7 +136,6 @@ namespace GeneratorFuelSupply {
                 UtilityRoutines::IsNameEmpty(state, AlphArray(1), cCurrentModuleObject, ErrorsFound);
 
                 state.dataGenerator->FuelSupply(FuelSupNum).Name = AlphArray(1);
-                ObjMSGName = cCurrentModuleObject + " Named " + AlphArray(1);
```

Comment:
[src/EnergyPlus/GeneratorFuelSupply.cc:155]:(style),[unreadVariable],Variable 'ObjMSGName' is assigned a value that is never used.